example/plugins/c-api/Makefile.am

```diff
@@ -14,8 +14,8 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-include $(top_srcdir)/build/plugins.mk
-include $(top_srcdir)/build/tidy.mk
+include $(top_srcdir)/m4/plugins.mk
+include $(top_srcdir)/m4/tidy.mk
```

Comment:
Should we put the makefile includes in `$(top_srcdir)/mk`?

Reply:
Great suggestion.